locale/cs.po

```diff
@@ -1334,6 +1334,15 @@ msgstr ""
 msgid "Polygon needs at least 3 points"
 msgstr ""
 
+#: ports/atmel-samd/common-hal/pulseio/PulseOut.c
+#: ports/cxd56/common-hal/pulseio/PulseOut.c
+#: ports/nrf/common-hal/pulseio/PulseOut.c
+#: ports/stm/common-hal/pulseio/PulseOut.c
+msgid ""
+"Port does not accept pins or frequency.                                     "
+"Construct and pass a PWMOut Carrier instead"
```

Comment:
@hierophect this looks weird, I think it's due to the way you split a long string across multiple lines inside translate().  

Does it work to do it in the more standard way, 
```
translate("Here is the first part of the string "
"and here is the second")
```
?  If not it's probably best to have an overly long line in the source rather than have these spaces in the messages.